ksqldb-serde/src/main/java/io/confluent/ksql/serde/protobuf/ProtobufNoSRConverter.java

```diff
@@ -44,6 +44,16 @@ public class ProtobufNoSRConverter implements Converter {
 
   private ProtobufData protobufData;
 
+  /**
+   * This constructor is defined only to adhere to the Converter plugin semantics of defining a
+   * no-arg constructor. The usage of this class is internal to ksqldb and is instantiated
+   * explicitly with Schema argument.
```

Comment:
> and is instantiated explicitly with Schema argument.

What does this mean? If ksqlDB uses ` ProtobufNoSRConverter(final Schema schema)` why do we need the non-arg constructor? Where is it uses?

Reply:
@mjsax - I have updated the documentation a bit. Have a look and let me know if it is clear for the reader.